openquake/commonlib/readinput.py

```diff
@@ -1640,6 +1649,18 @@ def read_countries_df(buffer=0.1):
     return read_geometries(fname, 'shapeGroup', buffer)
 
 
+def read_populated_places_df(lon_field='longitude', lat_field='latitude',
+                             label_field='name'):
+    """
+    :returns: a DataFrame of points corresponding to populated places
+    """
+    mosaic_dir = config.directory.mosaic_dir
+    fname = os.path.join(mosaic_dir, 'worldcities.csv')
```

Comment:
I wonder if we should put the content of worldcities.csv inside exposure.hdf5, so that we would depend on a single file.